include/FxMixerView.h

```diff
@@ -103,12 +103,14 @@ class EXPORT FxMixerView : public QWidget, public ModelView,
 	// useful for loading projects
 	void refreshDisplay();
 
+public slots:
+	int addNewChannel();
```

Comment:
I had to expose this method to use it on the Track view, is there a better approach?